pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[project]
+name = "jsonschema2md"
+version = "0.0.0"
+requires-python = ">=3.9,<4.0"
+description = "Convert JSON Schema to human-readable Markdown documentation"
+dependencies = [
+    "pyyaml==6.0.2",
+]
+
```

Comment:
Duplicated of Poetry section, I will apply properly the update to Poetry 2 later :-)

Reply:
Does poetry support this syntax? I would prefer a solution that is agnostic to the tooling used.

Reply:
Poetry 2 yes, I want to do it properly in another pull request :-)

Reply:
Great -- let me know if there is anything I can do to help.